website/src/docs/aws-s3.md

```diff
@@ -66,12 +66,13 @@ uppy.use(AwsS3, {
 
 Custom headers that should be sent along to [Companion][companion docs] on every request.
 
-### `metaFields: []`
+### `allowedMetaFields: null`
```

Comment:
Why did change the docs to say the default is now `null`, and kept `[]` in the code? Which one did we mean?